src/Components/RecommendationRating/RecommendationRating.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@patternfly/react-core';
+import { OutlinedThumbsUpIcon } from '@patternfly/react-icons';
+import { OutlinedThumbsDownIcon } from '@patternfly/react-icons';
+import { ThumbsUpIcon } from '@patternfly/react-icons';
+import { ThumbsDownIcon } from '@patternfly/react-icons';
+
+import './RecommendationRating.scss';
+import { ROS_API_ROOT, RECOMMENDATION_RATING_API } from '../../constants';
+
+const RecommendationRating = ({ system }) => {
+    const [rating, setRating] = useState(system.rating);
+    const [submitted, setSubmitted] = useState(false);
+    const updateRecommendationRating = async (newRating) => {
+        const calculatedRating = rating === newRating ? 0 : newRating;
```

Comment:
I'm not a huge fan of random numbers in code, do you think you could create 3 constants instead using -1/0/1?
```JS
const NEGATIVE_FEEDBACK = -1;
const NEUTRAL_FEEDBACK = 0;
const POSITIVE_FEEDBACK = 1;
```

Reply:
Done! I have added export statements above class component. Is it the right place to add?

Reply:
You can add some constants file to group all constants together, but I think it's good for now!

Reply:
We already have one constants file so moved these constants into same file.